src/interface/src/app/standalone/planning-areas/planning-areas.datasource.ts

```diff
@@ -8,21 +8,44 @@ import { QueryParamsService } from './query-params.service';
 export class PlanningAreasDataSource extends DataSource<PreviewPlan> {
   private readonly limit = 13;
 
-  private _dataStream = new Subject<PreviewPlan[]>();
+  private _dataStream = new BehaviorSubject<PreviewPlan[]>([]);
   private _loading = new BehaviorSubject(false);
+  private _hasFilters$ = new BehaviorSubject(false);
+  private _initialLoad$ = new BehaviorSubject(true);
 
   public count = 0;
   public pages = 0;
 
+  public sortOptions: Sort = this.queryParamsService.getInitialSortParams();
+  public pageOptions = this.queryParamsService.getInitialPageParams();
+  public searchTerm = this.queryParamsService.getInitialFilterParam();
+
+  /**
+   * Emits `true` if loading the first time or applying filters (where number of results change)
+   * `false` when done loading.
+   */
+  public initialLoad$ = this._initialLoad$.asObservable();
```

Comment:
@lastminutediorama what would you call this? Not sure `initialLoad` is clear enough

Reply:
It is a little confusing, but I can't think of another alternative. We can keep it for now, I think?